### DISCUSSION TITLE
Personal / Enterprise Co-Pilot Pricing Doesn't Make Sense

### DISCUSSION THREAD

**Discussion by ydax** [community/community#53136] [Enterprise]
### Select Topic Area

Product Feedback

### Body

I'm a developer and entrepreneur. One of my companies is a GitHub "organization", and our repo is hosted through the organization. Another one of my companies is run through a repo that's on a "personal" GitHub account. So, if I want to use co-pilot, I now have to take out two subscriptions:

1. An "enterprise seat" for my company that's a GitHub organization
2. A "personal seat" for my other company

However, I'm the same person using co-pilot on both. This is silly. This pricing structure needs to change. I should be able to pay for co-pilot for my account, ydax, and, when that account has this feature, the coder should be able to use it on any of their coding projects.

**Comment by yangm97** [comment]
The price jump between individual and business plan makes no sense either, given the only thing that changes between the two is the billing.

If you think about it, business plan should either be cheaper than personal or have more features.
